src/components/BeaconResults.vue

```diff
@@ -215,10 +215,44 @@ export default {
         this.response = [];
         this.notFound = true;
       }
+    },
+    setSearchToLocaStorage: function() {
+      if (localStorage.getItem("searches") == null) {
+        var searches = [];
+        var currentdate = new Date();
+        var time =
+          currentdate.getHours() +
+          ":" +
+          currentdate.getMinutes() +
+          ":" +
+          currentdate.getSeconds();
```

Comment:
pretty please can we also add the day, month and year :) because it is difficult to distinguish searches from today with the ones from yesterday.

For month we can use: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getMonth#using_getmonth `Intl.DateTimeFormat()`